### ISSUE TITLE
Add example markup for range limit labels

### ISSUE THREAD

**Issue by patrickcate** [uswds/uswds#4504]
**Is your feature request related to a problem? Please describe.**
There is usability guidance on the [range slider component](https://designsystem.digital.gov/components/range-slider/) page that states:

```
When appropriate, label the ends of the slider with the limits 
of the range (for example: “0/100”, “small/large” or 
“less expensive/more expensive”).
```

but there is no example markup on what the end labels should use. This makes it difficult to know how to add these labels so they display consistency, and in line with the USWDS. 

It is also unclear if there is accessibility concerns to using particular markup. For instance, are there `aria` attributes that should be included with this markup? 

**Describe the solution you'd like**
Add an example to the component documentation page of the end label markup.

**Comment by mgifford** [comment]
Looking at other examples:

- https://www.w3.org/WAI/ARIA/apg/patterns/slider/
- https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-color-viewer/
- https://www.w3.org/WAI/ARIA/apg/patterns/slider-multithumb/
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/slider_role
- https://www.a11ywithlindsey.com/blog/creating-accessible-range-slider-css
- https://dequeuniversity.com/library/aria/slider
- https://design.accor.com/latest/web/core-components/range-slider/overview-NppNNn0K
- https://github.com/tajo/react-range

Especially:

- https://carbondesignsystem.com/components/slider/accessibility/
- https://open-ui.org/components/enhanced-range-input.explainer/

The current range slider example is:

```
<form class="usa-form">
  <label class="usa-label" for="usa-range">Range slider</label>
  <input
    id="usa-range"
    class="usa-range"
    type="range"
    min=""
    max=""
    step=""
    value="20"
  />
</form>
```

This would at least set the range

```
<form class="usa-form">
  <label class="usa-label" for="usa-range">Range slider</label>
  <input
    id="usa-range"
    class="usa-range"
    type="range"
    min="0"
    max="100"
    step=""
    value="20"
  />
</form>
```

In the preview the aria-valuetext is dynamic, as is the span outside of it:

`<div class="usa-range__wrapper"><input id="usa-range" class="usa-range" type="range" min="" max="" step="" value="20" aria-valuetext="50 of 100" data-gtm-form-interact-field-id="0"><span aria-hidden="true" class="usa-range__value">50</span></div>`


The advice is indeed difficult to interpret based on the examples:

> When appropriate, label the ends of the slider with the limits of the range (for example: “0/100”, “small/large” or “less expensive/more expensive”).

I am also unsure why the preview and the example are inconsistent in their use of `<span aria-hidden="true" class="usa-range__value">50</span>`

I would like to see a better explanation for how `aria-valuetext="50 of 100"` is generated, but I haven't found an example elsewhere where the limits are expressed in a label like the example calls.  I think we should be improving that text. 

I would like to see the References updated too. Not sure these are the most accurate or useful now.

**Comment by ryparker** [comment]
The guidance recommends labels at each end of the range slider, but we're missing an example of how to add them. Let's add one tested example using the existing range input, covering `min`, `max`, and descriptive endpoint text without making screen readers repeat information the input already announces. This only needs a documentation change.
